module-system/module-implementations/sov-evm/src/call.rs

```diff
@@ -30,7 +30,7 @@ impl<C: sov_modules_api::Context> Evm<C> {
     ) -> Result<CallResponse> {
         let evm_tx_recovered: EvmTransactionSignedEcRecovered = tx.try_into()?;
 
-        let block_env = self.block_env.get(working_set).unwrap_or_default();
+        let block_env = self.pending_block.get(working_set).unwrap_or_default();
```

Comment:
Is the name ok here?

Reply:
I think so, it is data about the currently constructed/built block.

Reply:
So `pending_block` and `pending_transactions`